aspnetcore/tutorials/first-mvc-app/start-mvc.md

```diff
@@ -318,21 +371,19 @@ In the next part of this tutorial, you learn about MVC and start writing some co
 
 ::: moniker range="< aspnetcore-3.0"
 
-[!INCLUDE [consider RP](~/includes/razor.md)]
+This tutorial teaches ASP.NET Core MVC web development with controllers and views. If you're new to ASP.NET Core web development, consider the [Razor Pages](xref:tutorials/razor-pages/razor-pages-start) version of this tutorial, which provides an easier starting point.
```

Comment:
Why waste time on 3.1? If you're going to do that, use `::: moniker range="< aspnetcore-5.0"` given there's no code changes your showing.